src/define/Advert.ts

```diff
@@ -141,6 +142,12 @@ class Advert {
 		this.node = adSlotNode;
 		this.sizes = this.generateSizeMapping(additionalSizeMapping);
 
+		if (isEligibleForTeads(adSlotNode.id)) {
+			slotTargeting['teadsEligible'] = 'true';
+		} else {
+			slotTargeting['teadsEligible'] = 'false';
+		}
```

Comment:
Most other slot targeting is set in `defineSlot` might that be a better place to set this also?